.github/actions/run-publish-vr-screenshot/action.yml

```diff
@@ -0,0 +1,106 @@
+name: 'Run and Publish VR Screenshot'
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33070&commitId=9a5cf651fab61b2d5f3d47f82ccf8f8a8002a4c3&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Image.chromium.png|284| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>